### PR TITLE
Specification of polar bounding boxes

### LINKED ISSUE #47: Document best practice for polar bounding boxes
Anyone want to take a stab at documenting how to represent the bounding box of Antarctica in GeoJSON? @pramsey?

I'm finding http://www.jasondavies.com/maps/bounds/ to be a cool resource for thinking about this.

### PR DESCRIPTION
**Pull request by sgillies**
Closes #47

### CODE REVIEW

File: middle.mkd [hunk 1]
```diff
@@ -355,6 +355,24 @@ Example of a bbox for a line crossing the date-line:
...
+A bounding box that contains the North Pole extends from a southwest
+corner of <minlat>°N, 180°W to a northwest corner of 90°N, 180°E. Viewed
+on a globe, this bounding box approximates a spherical cap.
+
+    "bbox": [-180.0, <minlat>, 180.0, 90.0]
+
+A bounding box that contains the South Pole extends from a southwest
+corner of 90°S, 180°W to a northwest corner of <maxlat>°S, 180°E.
+
+    "bbox": [-180.0, <minlat>, 180.0, 90.0]
```
Comment (martinthomson) [inline middle.mkd]: [-180, -90, 180, <maxlat> ]

Also note that the degrees symbol (°) is not permitted in RFCs.  That's a relic of their ASCII heritage.  You might want to use * instead.

**Comment by martinthomson** [comment]
I think that the markdown translator has somehow neglected to translate `<maxlat>` as simple text.  You probably need to make this `&lt;maxlat>` or something like that.

**Comment by dr-shorthair** [comment]
Just checking: how do you specify the bounding box for New Zealand's Antarctic territory? It is a segment from 160E to 150W (i.e. crosses the anti-meridian), from the pole to 60S. I think it must be 

[160, -90, -150, 60] 

The lower-left position is a bit problematic as any longitude with -90 as the latitude is at the pole, but we also need to indicate the leftmost longitude ...

**Comment by martinthomson** [comment]
@dr-shorthair, that seems right to me.  It might be a valuable example of why a longitude on a polar coordinate is useful.  It might be a better example than the complete polar cap even.

**Pull request merged**